syslogdiag/emailing.py

```diff
@@ -65,21 +68,29 @@ def _send_msg(msg):
 
     """
 
-    email_server, email_address, email_pwd, email_to, email_port = get_email_details()
+    email_server, email_address, email_pwd, email_to, email_port, use_gmail = get_email_details()
 
     me = email_address
     you = email_to
     msg["From"] = me
     msg["To"] = you
 
+    if use_gmail is None:
```

Comment:
This is fucking horrible and won't work anyway since use_gmail won't be None if it's not present. Just add use_gmail: False to defaults.yaml.